api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ObservationReferenceRangeTranslatorImpl.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.translators.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.hl7.fhir.r4.model.CodeableConcept;
+import org.hl7.fhir.r4.model.Observation;
+import org.hl7.fhir.r4.model.Quantity;
+import org.openmrs.ConceptNumeric;
+import org.openmrs.module.fhir2.api.translators.ObservationReferenceRangeTranslator;
+
+public class ObservationReferenceRangeTranslatorImpl implements ObservationReferenceRangeTranslator {
```

Comment:
This class should have the annotations `@Component` and `@Setter(AccessLevel.PACKAGE)`. The first of those annotations registers this with Spring so it can be `@Inject`ed where appropriate. [A little more on that here](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/stereotype/Component.html). The second tells [Lombok](https://projectlombok.org/features/GetterSetter) to generate a setter.